src/main/java/org/broadinstitute/hellbender/tools/dataflow/transforms/markduplicates/ReadsKey.java

```diff
@@ -7,7 +7,7 @@
 /**
  * Encodes a unique key for read, read pairs and fragments. Used to identify duplicates for MarkDuplicates.
  */
-public final class MarkDuplicatesReadsKey {
+public final class ReadsKey {
```

Comment:
If this class is really only relevant to mark duplicates, it should maybe be made package protected.